it/controllers/cancelRegistration/CancelRegistrationDateControllerISpec.scala

```diff
@@ -19,8 +20,9 @@ class CancelRegistrationDateControllerISpec extends ControllerITTestHelper {
   val normalRoutePath = "/date"
   val checkRoutePath = "/change-date"
   val random = new Random()
-  val validCancellationDate = LocalDate.now().plusDays(random.nextLong(13))
-  val validCancellationDateJson = Json.obj(
+  val currentDate: String = LocalDate.now.format(DateTimeFormatter.ofPattern("d M yyyy"))
+  val validCancellationDate: LocalDate = LocalDate.now().plusDays(random.nextLong(13))
```

Comment:
Guessing there is already a test for more than 14 days in the future? (Just to check)

Reply:
Yes, in the Form spec tests